src/spirit-view/spirit-view.html

```diff
@@ -96,6 +96,12 @@
                     }
                 }
                 return true;
+            },
+
+            _removeName: function() {
```

Comment:
Currently if no name is entered it doesn't override the default name so that is still used, but this suggestion is better for the ux.